src/enum_base.hpp

```diff
@@ -82,9 +82,13 @@ struct enum_base : public T
 	}
 };
 
-#define ENUM_AND_ARRAY(...) \
-	static constexpr std::size_t count = std::tuple_size<decltype(std::make_tuple(__VA_ARGS__))>::value; \
-	enum class type { __VA_ARGS__ }; \
-	static constexpr std::array<const char*, count> values{ __VA_ARGS__ };
+#define ENUM_AND_ARRAY(...)                                                                                            \
+	enum class type { __VA_ARGS__ };                                                                                   \
+                                                                                                                       \
+	/** Provide a variable template for an array of matching size. */                                                  \
```

Comment:
This is not a variable template. A variable template is like `template<typename T> bool whatever = something<T>::value;`. Please fix the comment.

Reply:
Ah, you're right, I meant alias template. I removed `count` since I made `values` use the alias so I figured might as well subsume it.